### ISSUE TITLE
We shouldn't use access key

### ISSUE THREAD

**Issue by nickbristow** [18F/web-design-standards#1772]
https://standards.usa.gov/components/headers/#basic

There are some components with access key set on them in the sample code. Access key should be avoided as there are inconsistency in screen readers and this could override user shortcuts.

**Comment by juliaelman** [comment]
Quick note, talked with @nickbristow and we can remove this without having to replace with anything. Here are the spots that should be removed:

- src/templates/header-basic.html
- src/templates/header-extended.html
- src/templates/header-basic-mega.html
- src/templates/header-extended-mega.html 

We'll also need to remove from the [docs site repo](https://github.com/18F/web-design-standards-docs) in the following locations:

- _includes/navbar.html 
- _components/page-templates/template-docs.html 
- _components/page-templates/template-landing.html

**Comment by shawnbot** [comment]
Sad face! But good to know, and good to pass along to our users. Thanks, @nickbristow!

**Comment by juliaelman** [comment]
Completed in #1793 and https://github.com/18F/web-design-standards-docs/issues/206.
